fhir-server/liberty-config/server.xml

```diff
@@ -11,6 +11,7 @@
         <feature>jdbc-4.1</feature>
         <feature>websocket-1.1</feature>
         <feature>localConnector-1.0</feature>
+	<feature>monitor-1.0</feature>
```

Comment:
this is causing a merge conflict because server.xml has been moved in main.  lets remove it for now, merge, then add back in at the new location
```suggestion
```